plugins/inputs/procstat/sample.conf

```diff
@@ -43,3 +43,32 @@
   ## the native finder performs the search directly in a manor dependent on the
   ## platform.  Default is 'pgrep'
   # pid_finder = "pgrep"
+
+  ## Select which extra metrics should be added:
+  ##  - "threads": to enable collection of number of file descriptors
+  ##  - "fds": to enable collection of context switches
+  ##  - "ctx_switches": to enable collection of page faults
+  ##  - "page_faults": to enable collection of IO
+  ##  - "io": to enable collection of proc creation time
+  ##  - "create_time": to enable collection of CPU time used
+  ##  - "cpu": to enable collection of percentage of CPU used
+  ##  - "cpu_percent": to enable collection of memory used
+  ##  - "mem": to enable collection of memory percentage used
+  ##  - "mem_percent": to enable collection of procs' limits
+  ##  - "limits": to enable collection of procs' limits
+  ##  - "tcp_stats": tcp_* and upd_socket metrics
+  ##  - "connections_endpoints": new metric procstat_tcp with connections and listeners endpoints
+  ## Default value:
+  # metrics_include = [
+  #  "threads",
+  #  "fds",
+  #  "ctx_switches",
+  #  "page_faults",
+  #  "io",
+  #  "create_time",
+  #  "cpu",
+  #  "cpu_percent",
+  #  "mem",
+  #  "mem_percent",
+  #  "limits",
+  # ]
```

Comment:
Please keep this in one line so we keep the (already to large) config more compact.